src/Classes/MyRadio/MyRadioFormField.php

```diff
@@ -784,7 +784,7 @@ private function convertTime($timeString)
          * YES, PHP IS RETARDED.
          */
         $timeString = str_replace('/', '-', $timeString);
-        $time = (int) strtotime($timeString) - date('Z');
+        $time = (int) strtotime($timeString);
```

Comment:
No, the aim is for all server-side business logic to be UTC. I don't think this should be changed.